### PR TITLE
Fix deprecated Twig_Function_Method usage

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Tests fixed.

File: Tests/Twig/GravatarExtensionTest.php [hunk 1]
```diff
@@ -34,13 +45,16 @@ public function testName()
...
-        $this->assertEquals(array(
+        $expectedNames = array(
             'gravatar',
             'gravatar_hash',
             'gravatar_exists',
-        ), array_keys($this->extension->getFunctions()));
+        );
 
+        foreach ($expectedNames as $n =>$expectedName) {
+            $this->assertSame($expectedName, $this->extension->getFunctions()[$n]->getName());
```
Comment (henrikbjorn) [inline Tests/Twig/GravatarExtensionTest.php]: This makes the 5.3 test fail.

``` php
$functions = $this->extension->getFunctions();

$this->assertSame()
```

Also get the functions outside the loop.

**Comment by Soullivaneuh** [comment]
@henrikbjorn Should be OK now. :+1:

**Comment by henrikbjorn** [comment]
Thanks

**Comment by Soullivaneuh** [comment]
You're welcome.

Any chance to have a new patch tag soon?

**Comment by henrikbjorn** [comment]
1.1.3 tagged.

**Comment by Soullivaneuh** [comment]
:+1:

**Comment by Soullivaneuh** [comment]
Seems to not be updated on packagist: https://packagist.org/packages/ornicar/gravatar-bundle

**Comment by henrikbjorn** [comment]
I updated the hook and changed the github url on packagist.

**Comment by Soullivaneuh** [comment]
Works like a charm, thank you. :+1: 

Note: If you need help maintaining this project, I'll glad to be part of contributors. ;-)

**Pull request merged**